Engine/src/Engine/Rendering/Renderer.h

```diff
@@ -20,6 +21,8 @@ namespace Engine
 		glm::vec3 screenToWorld(const glm::vec2& screenPosition);
 		glm::vec2 worldToScreen(const glm::vec3& worldPosition);
 
+		AnimationSystem animationSystem;
```

Comment:
Is this used anywhere?

Reply:
It's used in Renderer.cpp ~line 160 to update the animation data of animation and lerp components. Declared public because scripting system may want to access it to interact with animations.